src/declaration.d

```diff
@@ -1357,6 +1357,25 @@ extern (C++) class VarDeclaration : Declaration
             storage_class &= ~stc; // strip off
         }
 
+        if (storage_class & STCscope)
+        {
+            StorageClass stc = storage_class & (STCstatic | STCextern | STCmanifest | STCtls | STCgshared);
+            if (stc)
+            {
+                OutBuffer buf;
+                stcToBuffer(&buf, stc);
+                error("cannot be scope and %s", buf.peekString());
```

Comment:
In many error messages keywords like `return` are surrounded in single quotes. That's why I think the message should be `cannot be 'scope' and '%s'`.